packages/bff/src/routers/authorizationRouter.ts

```diff
@@ -53,7 +56,25 @@ const authorizationRouter = (
         return res.status(err.status).send();
       }
     })
-    .post("/support", async (_req, res) => res.status(501).send());
+    .post("/support", async (req, res) => {
+      const saml = Buffer.from(req.body.SAMLResponse, "base64").toString();
```

Comment:
should this be inside the `try`, in case the given string is not a base64 value?